AnnoyancesFilter/sections/cookies_specific.txt

```diff
@@ -92,6 +92,9 @@ techadvisor.co.uk,forum.notebookreview.com,computerworld.com,skysports.com#$#bod
 !
 !
 lazimbana.com##.lazim-cookie-alert
+goldenline.pl#$#body{ overflow: visible !important; padding-right: 0 !important; }
+goldenline.pl#$?#.cookie-modal:upward(div.gl-modal) { display: none !important; }
+goldenline.pl#@#.cookie-modal
```

Comment:
`goldenline.pl#@#.cookie-modal` for `cookies_whitelist.txt` with task comment.

Reply:
Okay.